<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearchWidget</class>
 <widget class="QWidget" name="AdvancedSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Search</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="simpleTab">
      <attribute name="title">
       <string>Simple</string>
      </attribute>
      <layout class="QVBoxLayout" name="simpleLayout">
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Search Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo"/>
       </item>
       <item>
        <widget class="QLabel" name="queryLabel">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="queryEdit">
         <property name="placeholderText">
          <string>Enter search terms...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="limitLabel">
         <property name="text">
          <string>Results per page:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="limitSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="simpleVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="advancedLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>376</width>
            <height>493</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="scrollLayout">
           <item>
            <widget class="QLabel" name="advancedLabel">
             <property name="text">
              <string>Advanced search fields will appear here based on the selected entity type.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="advancedVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="buttonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
